includes/class-comment.php

```diff
@@ -264,17 +264,15 @@ public static function object_id_to_comment( $id ) {
 			array(
 				'meta_key'   => 'source_id', // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_key
 				'meta_value' => $id,         // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_value
+				'orderby'    => 'comment_date',
+				'order'      => 'DESC',
 			)
 		);
 
 		if ( ! $comment_query->comments ) {
 			return false;
 		}
 
-		if ( count( $comment_query->comments ) > 1 ) {
```

Comment:
We could probably just combine with previous check to be

```php
if ( count( $comment_query->comments ) !== 1 ) {
	return false
}
```

Reply:
the thing is: it does not make sense to check for the number, because if there is at least one (or more), it should return the first one.